src/scripts/main.js

```diff
@@ -1,3 +1,31 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promis1 = new Promise(function(resolve) {
+  logo.addEventListener('click', () => {
+    resolve('Promise was resolved!');
+  });
+});
+
+promis1.then((result) => {
+  const div = document.createElement('div');
+
+  div.classList.add('message');
+  div.innerText = result;
+  document.body.append(div);
```

Comment:
you repeated logic for append div with message - lets create common function and call it